src/Uuid/UuidGenerator.php

```diff
@@ -33,12 +33,12 @@ public function generateV4() {
      *
      * @return String   Valid v5 UUID.
      */
-    public function generateV5($str, $namespace = NULL) {
+    public function generateV5($namespace = NULL) {
         // Use default namespace if none is provided.
         if (empty($namespace)) {
             $namespace = $this->namespace;
         }
 
-        return Uuid::uuid5($namespace, $str)->toString();
+        return Uuid::uuid5(Uuid::NAMESPACE_DNS, $namespace)->toString();
```

Comment:
I checked through the Ramsey code because I was curious, and it looks like it takes care of that for us behind the scenes.